macros/edr/system/system_utils/empty_table.sql

```diff
@@ -20,6 +20,10 @@
     ]) }}
 {% endmacro %}
 
+{% macro empty_information_table() %}
+    {{ elementary.empty_table([('key', 'string'), ('value', 'string')]) }}
```

Comment:
cosmetics - I don't like having generic columns as it's less simple. I'd explicitly call them dbt_version, elementary_version

Reply:
I thought of something like [Snowflake's parameters](https://docs.snowflake.com/en/sql-reference/parameters.html) in which we'll store relatively generic information rather than solely the versions.